src/components/account/AssetOverview.tsx

```diff
@@ -142,9 +133,8 @@ export function AssetOverview({ asset, loading, refresh }: AssetOverviewProps) {
             {
               validator(_, value) {
                 setRecipient(value);
```

Comment:
Why do we need this validator if the result always passes?
If only in response to an event, use an event listener

Reply:
resolved